sanic/mixins/startup.py

```diff
@@ -308,7 +308,7 @@ def prepare(
         if fast:
             self.state.fast = True
             try:
-                workers = len(os.sched_getaffinity(0))
+                workers = len(os.sched_getaffinity(0)) # type: ignore
```

Comment:
Couldn't we just solve that above with:

```py
workers: int
...
```

Reply:
`sanic\mixins\startup.py:311: error: Module has no attribute "sched_getaffinity"`